x/blob/types/params.go

```diff
@@ -111,6 +112,7 @@ func validateMaxSquareSize(v interface{}) error {
 	return nil
 }
 
+// validateMinMaxSquareSizeOrder validates that minSquareSize if less than or equal to maxSquareSize
 func validateMinMaxSquareSizeOrder(minSquareSize, maxSquareSize uint32) error {
 	if minSquareSize > maxSquareSize {
 		return fmt.Errorf("max square size cannot be less than min square size")
```

Comment:
[no change needed] because not modified in this PR but proposal for the error message to match the ordering in the conditional so that it's easier to review.

```suggestion
	if minSquareSize > maxSquareSize {
		return fmt.Errorf("min square size cannot be greater than max square size")
```